src/client/common/types.ts

```diff
@@ -620,3 +623,16 @@ export interface IInterpreterPathService {
     update(resource: Resource, configTarget: ConfigurationTarget, value: string | undefined): Promise<void>;
     copyOldInterpreterStorageValuesToNew(resource: Uri | undefined): Promise<void>;
 }
+
+/**
+ * Interface used to retrieve the default language server to use when in experiment
+ *
+ * Note: This is added to get around a problem that the config service is not `async`.
+ * Adding experiment check there would mean touching the entire extension. For simplicity
+ * this is a solution.
+ */
+export const IDefaultLanguageServer = Symbol('IDefaultLanguageServer');
+
+export interface IDefaultLanguageServer {
+    readonly defaultLSType: LanguageServerType;
```

Comment:
```suggestion
    readonly defaultLSType: LanguageServerType.Jedi | LanguageServerType.JediLSP;
```

With how many times `LanguageServerType.Jedi | LanguageServerType.JediLSP` is used, a type alias would be welcome (but not necessary).

Reply:
We discussed this offline. No need to change this for now.